src/main/java/com/deavensoft/paketomat/user/UserController.java

```diff
@@ -24,48 +24,72 @@ public class UserController {
 
     private UserService userService;
 
+    private UserServiceImpl userServiceImpl;
+    @Autowired
```

Comment:
- [ ] Remove

Reply:
If I delete that annotation, when I want to get all the users, I get an error that reads:
_Cannot invoke "com.deavensoft.paketomat.mapper.UserMapper.userToUserDTO(com.deavensoft.paketomat.center.model.User)" because "this.userMapper" is null_

Reply:
Do you delete UserServiceImpl?

Reply:
Yes, it's deleted